src/archivist/errors.js

```diff
@@ -1,9 +1,7 @@
 export class InaccessibleContentError extends Error {
   constructor(message) {
```

Comment:
If `message` can be a string or an array of strings, I guess it should be called “reasonOrReasons”.